### PR TITLE
Revamp ITR documentation

### PR DESCRIPTION
**Pull request by edrevo**
### What does this PR do?
Revamp the ITR documentation:
- Include more content in the landing page for ITR, removing things that do not apply anymore and highlighting what the product does.
- Split out language setup to their own pages
- Add link to ITR from Test Visibility setup 
- Minimized the instructions for ITR setup to include only the ITR-specific stuff. For example, if users want to user agentless or the agent there's already better documentation about that in the test visibility setup.
- Fixed some command line bugs in the JS instructions.

### Motivation
@andrea-mosk told me our current doc was a bit depressing. I agree.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/intelligent_test_runner/javascript.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+## Compatibility
+
+Intelligent Test Runner is only supported in the following versions and testing frameworks:
+
+* `jest>=24.8.0`
+  * From `dd-trace>=3.10.0` or `dd-trace>=2.23.0`
```
Comment (edrevo) [inline content/en/continuous_integration/intelligent_test_runner/javascript.md:20]: Need confirmation from @juan-fernandez about these

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 2]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
 ---
-{{< callout url="https://app.datadoghq.com/ci/getting-started" >}}
-Intelligent Test Runner for CI Visibility is in private beta. You can request access by completing the form on the CI Visibility Getting Started page.
-{{< /callout >}}
 
-## Overview
+<div class="alert alert-warning">Intelligent Test runner is currently a beta product.</div>
 
-Intelligent Test Runner is Datadog's test impact analysis solution. It allows you to only run the impacted tests for a given commit and skip any that are irrelevant.
+Intelligent Test Runner is Datadog's test impact analysis solution. It automatically selects and runs only the relevant tests needed to validate any given commit, enabling you to reduce testing downtime while maintaining test coverage.
```
Comment (edrevo) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:16]: The text in this page I've basically copied from https://www.datadoghq.com/blog/streamline-ci-testing-with-datadog-intelligent-test-runner/

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 3]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
 [1]: /continuous_integration/tests/
-[2]: https://app.datadoghq.com/organization-settings/api-keys
-[3]: https://app.datadoghq.com/organization-settings/application-keys
-[4]: /getting_started/site/
-[5]: https://app.datadoghq.com/ci/settings/test-service
-[6]: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test
-[7]: https://docs.microsoft.com/en-us/visualstudio/test/vstest-console-options
+[2]: https://app.datadoghq.com/ci/settings/test-service
+[3]: https://app.datadoghq.com//ci/test-runs
+[4]: https://app.datadoghq.com/dash/integration/30941/ci-visibility-intelligent-test-runner-beta
```
Comment (edrevo) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:80]: Need to confirm this link with @drodriguezhdez

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 4]
```diff
@@ -10,187 +10,71 @@ further_reading:
     tag: "Blog"
     text: "Monitor all your CI pipelines with Datadog"
 ---
-{{< callout url="https://app.datadoghq.com/ci/getting-started" >}}
-Intelligent Test Runner for CI Visibility is in private beta. You can request access by completing the form on the CI Visibility Getting Started page.
-{{< /callout >}}
 
-## Overview
+<div class="alert alert-warning">Intelligent Test runner is a beta product.</div>
```
Comment (andrea-mosk) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:14]:
```suggestion
<div class="alert alert-warning">Intelligent Test Runner is a beta product.</div>
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md (hunk 2 above)
Comment (KickingTheTV) [reply content/en/continuous_integration/intelligent_test_runner/_index.md:16]: "Test Impact Analysis" is the industry term for what we're doing, the analysis of which tests are impacted. I think it's helpful to include in terms of SEO (Search engine optimization). Datadog docs pages rank quite high with google, so hopefully when someone searches for "Test impact analysis" our docs page can show up.

I also agree that "testing downtime" is unclear/misleading.

What do you think about this combination of suggestions:

"Intelligent Test Runner is Datadog's test impact analysis solution. It automatically and intelligently selects and runs only the relevant tests for a given commit based on the code being changed. Significantly reduce time spent testing and overall CI costs, while maintaining test coverage."

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 5]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
+When Intelligent Test Runner is active and skipping tests, you can see a purple text which displays the amount of time you saved on each test session or on each commit. The duration bar also changes color to purple so you can quickly identify which test sessions were using Intelligent Test Runner in the [Test Runs][3] page.
 
-## Configuration
+## Explore adoption and global savings
 
-The default branch of your repository is automatically excluded from having Intelligent Test Runner enabled. Due to the limitations described above, the Intelligent Test Runner might skip some of the tests that should be run, therefore Datadog recommends to continue running all tests in your default branch (or the branch you release from).
+You can also track you organization's savings and adoption of Intelligent Test Runner through our out-of-the-box [Intelligent Test Runner dashboard][4], which includes widgets to track your overall savings as well as a per-repository, per-committer and per-service view of the data so you can understand where and who is getting the most out of Intelligent Test Runner.
```
[the remark this answers is not in the document]
Comment (KickingTheTV) [reply content/en/continuous_integration/intelligent_test_runner/_index.md]: I would change this last sentence to something more like:

"...so you can understand what parts of your organization are using and getting the most out of Intelligent Test Runner."

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 6]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
 ---
-{{< callout url="https://app.datadoghq.com/ci/getting-started" >}}
-Intelligent Test Runner for CI Visibility is in private beta. You can request access by completing the form on the CI Visibility Getting Started page.
-{{< /callout >}}
 
-## Overview
+<div class="alert alert-warning">Intelligent Test Runner is a beta product.</div>
 
-Intelligent Test Runner is Datadog's test impact analysis solution. It allows you to only run the impacted tests for a given commit and skip any that are irrelevant.
+Intelligent Test Runner is Datadog's test impact analysis solution. It automatically and intelligently selects and runs only the relevant tests for a given commit based on the code being changed. Significantly reduce time spent testing and overall CI costs, while maintaining test coverage.
```
Comment (juan-fernandez) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:16]: "automatically and intelligently" sounds more like marking material than docs. That's just my opinion though

Comment (juan-fernandez) [reply content/en/continuous_integration/intelligent_test_runner/_index.md:16]: I'm sure the docs team will come up with something better but I'd rather do something like

> It uses the code changed in a given commit to automatically select and run only the relevant tests.

```suggestion
Intelligent Test Runner is Datadog's test impact analysis solution. It uses the code changed in a given commit to automatically select and run only the relevant tests. It significantly reduces time spent testing and overall CI costs, while maintaining test coverage.
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 7]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
 
-By only running tests on relevant code, when tests fail, it is likely a legitimate failure that pertains to the modified code.
+It does this by analyzing your test suite to determine the code each test covers, and then cross-referencing that coverage with the files impacted by a new code change. Datadog uses this information to run a selection of relevant, impacted tests, omitting the ones unaffected by the code change and reducing the overall testing duration.
 
-## Limitations during beta
+By minimizing the number of tests run per commit, Intelligent Test Runner reduces the frequency of flaky tests disrupting your pipelines. Flaky tests are tests that may pass or fail at random given the same commit. This can be particularly frustrating when the test flaking is unrelated to the code change being tested. After enabling Intelligent Test Runner for your test services, you can limit each commit to its relevant tests to ensure that flaky tests unrelated to your code change don’t end up arbitrarily breaking your build.
 
-During the beta of Intelligent Test Runner there are certain limitations:
+### Limitations during beta
```
Comment (juan-fernandez) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:25]: is that a limitation only during the beta? We might want to set expectations here

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 8]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
-`DD_TEST_RUNNER`
-: Enables or disables the instrumentation of tests. Set this value to `$(DD_TEST_RUNNER)` so you can enable and disable test instrumentation with a environment variable defined outside of the test process (for example, in the CI build).<br/>
-**Default**: `false`<br/>
-**Recommended**: `$(DD_TEST_RUNNER)`
+{{< img src="continuous_integration/itr_overview.png" alt="Intelligent test runner enabled in test service settings in the CI section of Datadog." style="width:80%;">}}
 
-`DD_API_KEY` (Required)
-: The [Datadog API key][2] used to upload the test results.<br/>
-**Default**: `(empty)`
+Due to the limitations described above, the default branch of your repository is automatically excluded from having Intelligent Test Runner enabled. This is the recommended configuration by Datadog to ensure that all of your tests are run in the default branch.
```
Comment (juan-fernandez) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:47]: the last phrase does not explain much, does it? 
Maybe 
```suggestion
Due to the limitations described above, the default branch of your repository is automatically excluded from having Intelligent Test Runner enabled. This is the recommended configuration by Datadog to minimize the risks of false positives. 
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 9]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
 
-`DD_APPLICATION_KEY` (Required)
-: The [Datadog Application key][3] used to query the tests to be skipped.<br/>
-**Default**: `(empty)`
+If there are other branches you want to exclude, you can add them from the Intelligent Test Runner settings page. The query bar supports the wildcard character `*` to exclude any branches that match.
```
Comment (juan-fernandez) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:49]: maybe add an example? 
```suggestion
If there are other branches you want to exclude, you can add them from the Intelligent Test Runner settings page. The query bar supports the wildcard character `*` to exclude any branches that match, such as `release_*`
```

File: content/en/continuous_integration/intelligent_test_runner/javascript.md (hunk 1 above)
Comment (juan-fernandez) [inline content/en/continuous_integration/intelligent_test_runner/javascript.md:20]: let's set them to these versions, as they include a bug fix
```suggestion
  * From `dd-trace>=3.11.0` or `dd-trace>=2.24.0`
```

File: content/en/continuous_integration/intelligent_test_runner/javascript.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+  * Only `jest-circus/runner` is supported as `testRunner`.
+  * Only `jsdom` and `node` are supported as test environments.
+* `mocha>=5.2.0`
+  * From `dd-trace>=3.10.0` or `dd-trace>=2.23.0`
```
Comment (juan-fernandez) [inline content/en/continuous_integration/intelligent_test_runner/javascript.md:24]:
```suggestion
  * From `dd-trace>=3.11.0` or `dd-trace>=2.24.0`
```

**Review by juan-fernandez — APPROVED** [review]
left some minor comments but I like the revamp. Thanks for this!

File: content/en/continuous_integration/intelligent_test_runner/_index.md (hunk 6 above)
Comment (edrevo) [reply content/en/continuous_integration/intelligent_test_runner/_index.md:16]: I see this initial text is contentious! Haha! There's already a thread about this in https://github.com/DataDog/documentation/pull/17286#discussion_r1135944042

@KickingTheTV, I think you have the last word here.

**Review by andrea-mosk — APPROVED** [review]
Thanks a lot for the effort!

File: content/en/continuous_integration/intelligent_test_runner/_index.md (hunk 8 above)
Comment (edrevo) [reply content/en/continuous_integration/intelligent_test_runner/_index.md]: Discussed offline and rephrased.

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 11]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
-- There are known limitations in the current implementation of Intelligent Test Runner that can cause it to skip tests that should be run. Intelligent Test Runner is not able to detect:
-  - Changes in library dependencies.
-  - Changes in compiler options.
-  - Changes in external services.
-  - Changes to data files in data-driven tests.
+There are known limitations in the current implementation of Intelligent Test Runner that can cause it to skip tests that should be run under certain conditions. Specifically, Intelligent Test Runner is not able to detect changes in library dependencies, compiler options, external services or changes to data files in data-driven tests.
 
 To override Intelligent Test Runner and run all tests, add `ITR:NoSkip` (case insensitive) anywhere in your Git commit message.
 
 ## Setup Datadog Library
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:31]:
```suggestion
## Set up Datadog library
```
Let's take the opportunity to bring this heading into compliance with our capitalization and wording standards.

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 12]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
 
-Prior to setting up Intelligent Test Runner, you must have finished setting up [Test Visibility][1] for your particular language.
+Prior to setting up Intelligent Test Runner, you must have finished setting up [Test Visibility][1] for your particular language. If you are reporting data through the Datadog Agent you need to use v6.40+/v7.40+.
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:33]:
```suggestion
Prior to setting up Intelligent Test Runner, you must set up [Test Visibility][1] for your particular language. If you are reporting data through the Agent, use v6.40+/v7.40+.
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 13]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
-
-In addition to setting the environment variables, you or a user in your organization with admin permissions must activate the Intelligent Test Runner on the [Test Service Settings][5] page.
-
-### Swift
-
-To enable Intelligent Test Runner, the version of the `dd-sdk-swift` framework must be >= 2.2.0. The `Code Coverage` option must also be enabled in the test settings of your scheme or test plan, or  `--enable-code-coverage` must be added to your `swift test` command (if using a SPM target).
+## Configuration
 
-The following environment variables must also be set:
+Once you have setup your Datadog Library for Intelligent Test Runner, you can configure it from the [Test Service Settings][2] page.
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:43]:
```suggestion
Once you have setup your Datadog library for Intelligent Test Runner, configure it from the [Test Service Settings][2] page.
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 14]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
-`DD_TEST_RUNNER`
-: Enables or disables the instrumentation of tests. Set this value to `$(DD_TEST_RUNNER)` so you can enable and disable test instrumentation with a environment variable defined outside of the test process (for example, in the CI build).<br/>
-**Default**: `false`<br/>
-**Recommended**: `$(DD_TEST_RUNNER)`
+{{< img src="continuous_integration/itr_overview.png" alt="Intelligent test runner enabled in test service settings in the CI section of Datadog." style="width:80%;">}}
 
-`DD_API_KEY` (Required)
-: The [Datadog API key][2] used to upload the test results.<br/>
-**Default**: `(empty)`
+Due to the limitations described above, the default branch of your repository is automatically excluded from having Intelligent Test Runner enabled. This is the recommended configuration by Datadog which ensures that all of your tests are run prior to reaching production.
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:47]:
```suggestion
Due to the limitations described above, the default branch of your repository is automatically excluded from having Intelligent Test Runner enabled. Datadog recommends this configuration to ensure that all of your tests run prior to reaching production.
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 15]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
 
-`DD_APPLICATION_KEY` (Required)
-: The [Datadog Application key][3] used to query the tests to be skipped.<br/>
-**Default**: `(empty)`
+If there are other branches you want to exclude, you can add them from the Intelligent Test Runner settings page. The query bar supports the wildcard character `*` to exclude any branches that match, such as `release_*`
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:49]:
```suggestion
If there are other branches you want to exclude, add them from the Intelligent Test Runner settings page. The query bar supports using the wildcard character `*` to exclude any branches that match, such as `release_*`
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 16]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
-: The [Datadog site][4] to upload results to.<br/>
-**Default**: `datadoghq.com`<br/>
-**Selected site**: {{< region-param key="dd_site" code="true" >}}
+{{< img src="continuous_integration/itr_configuration.png" alt="Select branches to exclude from intelligent test runner" style="width:80%;">}}
 
-#### UI activation
+## Explore test sessions
 
-In addition to setting the environment variables, you or a user in your organization with admin permissions must activate the Intelligent Test Runner on the [Test Service Settings][5] page.
+You can explore the time savings you are getting from Intelligent Test Runner by looking at the test commit page and test sessions panel.
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:55]:
```suggestion
You can explore the time savings you get from Intelligent Test Runner by looking at the test commit page and test sessions panel.
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 17]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
 
-Intelligent Test Runner uses git metadata information (commit history) to work. However, some CI providers use a git shallow clone (`git clone --depth=0`) which only downloads the target commit without downloading any historical commit information. This setup does not contain enough information for Intelligent Test Runner to work. If your CI is using shallow clones, it must be changed.
+{{< img src="continuous_integration/itr_savings.png" alt="Intelligent test runner enabled in a test session showing its time savings." style="width:80%;">}}
 
-An efficient alternative to shallow clones are partial clones (supported in Git v2.27+), which will clone the current commit plus the necessary git metadata without retrieving all past versions of all files: `git clone --filter=blob:none`.
+When Intelligent Test Runner is active and skipping tests, you can see a purple text which displays the amount of time saved on each test session or on each commit. The duration bar also changes color to purple so you can quickly identify which test sessions are using Intelligent Test Runner in the [Test Runs][3] page.
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:61]:
```suggestion
When Intelligent Test Runner is active and skipping tests, purple text displays the amount of time saved on each test session or on each commit. The duration bar also changes color to purple so you can quickly identify which test sessions are using Intelligent Test Runner in the [Test Runs][3] page.
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 18]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
 
-## Configuration
+## Explore adoption and global savings
 
-The default branch of your repository is automatically excluded from having Intelligent Test Runner enabled. Due to the limitations described above, the Intelligent Test Runner might skip some of the tests that should be run, therefore Datadog recommends to continue running all tests in your default branch (or the branch you release from).
+You can also track you organization's savings and adoption of Intelligent Test Runner through the out-of-the-box [Intelligent Test Runner dashboard][4]. The dashboard includes widgets to track your overall savings as well as a per-repository, per-committer and per-service view of the data so you can understand what parts of your organization are using and getting the most out of Intelligent Test Runner.
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:65]:
```suggestion
Track you organization's savings and adoption of Intelligent Test Runner through the out-of-the-box [Intelligent Test Runner dashboard][4]. The dashboard includes widgets to track your overall savings as well as a per-repository, per-committer, and per-service view of the data. View the dashboard to understand which parts of your organization are using and getting the most out of Intelligent Test Runner.
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 19]
```diff
@@ -10,187 +10,71 @@ further_reading:
...
 
-If there are other branches you want to exclude, you can add them from the Intelligent Test Runner settings page. The query bar supports the wildcard character `*` to exclude any branches that match.
+{{< img src="continuous_integration/itr_dashboard1.png" alt="Intelligent Test Runner dashboard" style="width:80%;">}}
 
-{{< img src="continuous_integration/itr_configuration.png" alt="Select branches to exclude from intelligent test runner" style="width:80%;">}}
+It also allows you to track adoption of Intelligent Test Runner throughout your organization.
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:69]:
```suggestion
The dashboard also tracks adoption of Intelligent Test Runner throughout your organization.
```

File: content/en/continuous_integration/intelligent_test_runner/dotnet.md [hunk 20]
```diff
@@ -0,0 +1,67 @@
...
+      text: "Troubleshooting CI"
+---
+
+## Compatibility
+
+Intelligent Test Runner is only supported on `dd-trace>= 2.22.0` (execute `dd-trace --version` to get the version of the tool).
+
+## Setup
+
+Prior to setting up Intelligent Test Runner, you must have finished setting up [Test Visibility for .NET][1]. If you are reporting data through the Datadog Agent you need to use v6.40+/v7.40+.
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/dotnet.md:21]:
```suggestion
Prior to setting up Intelligent Test Runner, set up [Test Visibility for .NET][1]. If you are reporting data through the Agent, use v6.40+/v7.40+.
```

File: content/en/continuous_integration/intelligent_test_runner/dotnet.md [hunk 21]
```diff
@@ -0,0 +1,67 @@
...
+
+To enable Intelligent Test Runner, the following environment variables need to be set:
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/dotnet.md:23]:
```suggestion
To enable Intelligent Test Runner, set the following environment variables:
```

File: content/en/continuous_integration/intelligent_test_runner/javascript.md [hunk 22]
```diff
@@ -0,0 +1,62 @@
...
+* `jest>=24.8.0`
+  * From `dd-trace>=3.11.0` or `dd-trace>=2.24.0`
+  * Only `jest-circus/runner` is supported as `testRunner`.
+  * Only `jsdom` and `node` are supported as test environments.
+* `mocha>=5.2.0`
+  * From `dd-trace>=3.11.0` or `dd-trace>=2.24.0`
+
+## Setup
+
+Prior to setting up Intelligent Test Runner, you must have finished setting up [Test Visibility for Javascript and Typescript][1]. If you are reporting data through the Datadog Agent you need to use v6.40+/v7.40+.
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/javascript.md:28]:
```suggestion
Prior to setting up Intelligent Test Runner, set up [Test Visibility for Javascript and Typescript][1]. If you are reporting data through the Agent, use v6.40+/v7.40+.
```

File: content/en/continuous_integration/intelligent_test_runner/javascript.md [hunk 23]
```diff
@@ -0,0 +1,62 @@
...
+
+To enable Intelligent Test Runner, the following environment variables need to be set:
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/javascript.md:30]:
```suggestion
To enable Intelligent Test Runner, set the following environment variables:
```

File: content/en/continuous_integration/intelligent_test_runner/swift.md [hunk 24]
```diff
@@ -0,0 +1,49 @@
...
+      text: "Troubleshooting CI"
+---
+
+## Compatibility
+
+Intelligent Test Runner is only supported on `dd-sdk-swift>= 2.2.0`.
+
+## Setup
+
+Prior to setting up Intelligent Test Runner, you must have finished setting up [Test Visibility for Swift][1]. The Code Coverage option must also be enabled in the test settings of your scheme or test plan, or --enable-code-coverage must be added to your swift test command (if using a SPM target).
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/swift.md:21]:
```suggestion
Prior to setting up Intelligent Test Runner, set up [Test Visibility for Swift][1]. The **code coverage** option must also be enabled in the test settings of your scheme or test plan, or --enable-code-coverage must be added to your Swift test command (if using a SPM target).
```

File: content/en/continuous_integration/intelligent_test_runner/swift.md [hunk 25]
```diff
@@ -0,0 +1,49 @@
...
+
+If you are reporting data through the Datadog Agent you need to use v6.40+/v7.40+.
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/swift.md:23]:
```suggestion
If you are reporting data through the Agent, use v6.40+/v7.40+.
```

File: content/en/continuous_integration/intelligent_test_runner/swift.md [hunk 26]
```diff
@@ -0,0 +1,49 @@
...
+
+To enable Intelligent Test Runner, the following environment variables need to be set:
```
Comment (urseberry) [inline content/en/continuous_integration/intelligent_test_runner/swift.md:25]:
```suggestion
To enable Intelligent Test Runner, set the following environment variables:
```

**Comment by edrevo** [comment]
Many thanks for the review @urseberry! Just FYI, there are 10 instances of `Datadog Library` (with that capitalization) throughout the documentation (not just this one, but all of Datadog's docs), in case you want to open a PR to normalize that.

**Review by urseberry — APPROVED** [review]
Found one more small nit. Approving so I don't hold up the PR.

**Comment by edrevo** [comment]
`dd-gitlab/set_redirect_metadata_preview` seems to have failed but there's no link to gitlab to check what went wrong. do you know what could have cause it to fail @urseberry?

**Comment by urseberry** [comment]
> `dd-gitlab/set_redirect_metadata_preview` seems to have failed but there's no link to gitlab to check what went wrong. do you know what could have cause it to fail @urseberry?

Here is the link: https://gitlab.ddbuild.io/DataDog/documentation/-/jobs/241224642
The job is related to some work the Websites team is doing in the documentation repo. It is unrelated to your PR.

I'll go ahead and merge this. Thanks again!

**Pull request merged**